File: mtgSearch/src/main/java/com/dbottillo/mtgsearchfree/view/fragments/ReleaseNoteFragment.java
package com.dbottillo.mtgsearchfree.view.fragments;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.TextView;

import com.dbottillo.mtgsearchfree.R;
import com.dbottillo.mtgsearchfree.ui.BasicFragment;

import butterknife.BindView;
import butterknife.ButterKnife;

public class ReleaseNoteFragment extends BasicFragment {

    @BindView(R.id.release_note)
    TextView releaseNote;

    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
        View view = inflater.inflate(R.layout.fragment_release_note, container, false);
        ButterKnife.bind(this, view);
        return view;
    }

    public void onViewCreated(View view, Bundle savedInstanceState) {
        super.onViewCreated(view, savedInstanceState);
        setActionBarTitle(getString(R.string.action_release_note));
        releaseNote.setText(getText(R.string.release_note_text_full));
    }

    public String getPageTrack() {
        return "/release-note";
    }

}

File: mtgSearch/src/main/kotlin/com/dbottillo/mtgsearchfree/view/fragments/ReleaseNoteFragment.kt
package com.dbottillo.mtgsearchfree.view.fragments

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.TextView
import butterknife.ButterKnife
import butterknife.bindView
import com.dbottillo.mtgsearchfree.R

class ReleaseNoteFragment : BasicFragment() {

    val releaseNote: TextView by bindView(R.id.release_note)

    override fun onCreateView(inflater: LayoutInflater?, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        val view = inflater!!.inflate(R.layout.fragment_release_note, container, false)
        ButterKnife.bind(this, view)
        return view
    }

    override fun onViewCreated(view: View?, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)
        setActionBarTitle(getString(R.string.action_release_note))
        releaseNote.text = getText(R.string.release_note_text_full)
    }

    override fun getPageTrack(): String? {
        return "/release-note"
    }

}
